src/gc/gcinterface.h

```diff
@@ -546,16 +546,16 @@ class IGCHeap {
     virtual HRESULT Initialize() = 0;
 
     // Returns whether nor this GC was promoted by the last GC.
-    virtual BOOL IsPromoted(Object* object) = 0;
+    virtual bool IsPromoted(Object* object) = 0;
 
     // Returns true if this pointer points into a GC heap, false otherwise.
-    virtual BOOL IsHeapPointer(void* object, BOOL small_heap_only = FALSE) = 0;
+    virtual bool IsHeapPointer(void* object, bool small_heap_only = false) = 0;
```

Comment:
it's unfortunate that we don't see the warning when we call these methods 'cause we are converting BOOL to bool. if you turn the build output to verbose level, perhaps somewhere you can see C4800 is disabled (not necessarily in code, it could also just be in some build scripts).